src/packet/packet.c

```diff
@@ -0,0 +1,42 @@
+/**
+ * @author  @developStorm
+ * @date    2025-05-17
+ *
+ * Implementation of packet authentication methods.
+ */
+
+#include "packet.h"
+#include "logger.h"
+
+#include <tinycrypt/hmac.h>
+#include <tinycrypt/sha256.h>
+#include <tinycrypt/utils.h>
+
+#define PACKET_HMAC_PSK "StanfordSSIUMoJBTXpnXoZX3bQrNm5f"
```

Comment:
As previously mentioned in an earlier comment, have this be a CMake option definition so we can pass it in during build time instead of exposing it in our repo.

Reply:
fixed in 7bac89e748b56738e42d99497c35feceb72757f7